app/components/select-bestuurseenheid.js

```diff
@@ -1,46 +1,51 @@
 import Component from '@ember/component';
 import { inject as service } from '@ember/service';
-import { task, timeout } from 'ember-concurrency';
+import { action } from '@ember/object';
+import { timeout } from 'ember-concurrency';
+import { task } from 'ember-concurrency-decorators';
 
-export default Component.extend({
-  store: service(),
+export default class SelectBestuurseenheid extends Component {
```

Comment:
To convert this component to a Glimmer component, you can have a look at https://github.com/lblod/frontend-toezicht-abb/blob/master/app/components/filter/administrative-unit-select.js for example